flake.nix

```diff
@@ -451,9 +456,43 @@
       );
       devShell = perSystem (system: self.flake.${system}.devShell);
 
-      nixCi = flake-compat-ci.lib.recurseIntoFlakeWith {
+      effects = { src }:
+        let
+          pkgs = nixpkgs.legacyPackages.x86_64-linux;
+          nix = (builtins.getFlake "github:matthewcroughan/nixcfg/f16df936ffc9850d42f68f4badef9b55d7e7fee8").nixosConfigurations.swordfish.pkgs.nixFlakes;
```

Comment:
Why do we need `github:matthewcroughan/nixcfg`? Isn't the nix executable available from the `pkgs` in line 461?

Reply:
Ah, I left that in by mistake, this was something I was debugging.